Utils/azure-explorer-common/test/java/com/microsoft/tooling/msservices/serviceexplorer/azure/rediscache/RedisCacheIntegrationTest.java

```diff
@@ -70,7 +71,7 @@
     SubscriptionManager.class,
     DefaultLoader.class
 })
-
+@Ignore
```

Comment:
Ignore this test for now because the requests/response records are out-of-date. Tracked by https://github.com/Microsoft/azure-tools-for-java/issues/1883